caiman/source_extraction/volpy/atm.py

```diff
@@ -374,6 +375,8 @@ def test_spikeshape(time, tcourse, tcourse_med, tcourse_std, regress_matrix):
         # iteratively remove too small spikes 'slowly' increasing threshold
         for thresh in threshs:
             while np.sum(spikesizes < thresh):
+                import pdb
```

Comment:
Consider importing this at the head of the file rather than dynamically. Also, did you mean to leave this in?

Reply:
Sorry this is my mistake. atm.py file should not be changed.